### DISCUSSION TITLE
Augmenting code for outputting on two separate strips

### DISCUSSION THREAD

**Discussion by knbchse** [MaJerle/stm32-ws2811-ws2812-ws2812b-ws281x-tim-pwm-dma-timer#4] [Q&A]
Hi there,

Thanks for sharing your code @MaJerle! 
What is the process for augmenting this code to output data on another pin for a separate strip and having the two data streams separate?

Would this involve setting up another timer and DMA stream? Would you be able to provide some advice/help?

CHeers

**Comment by MaJerle** [comment]
In theory, you need one timer channel for one strip sequence. It also means you may use a timer with more channels to maximize utilization of your resources. For example TIM2 has 4 channels.

This is an example only, maybe having further expansions isn't a bad idea.
